tests/drivers/flash/boards/nrf52840dk_mx25l51245g.overlay

```diff
@@ -0,0 +1,37 @@
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &mx25r64;
+
+&qspi {
+	sck-pin = <30>;
+	io-pins = <29>, <28>, <04>, <03>;
+	csn-pins = <31>;
```

Comment:
This board uses PINCTRL now, so you need to rebase and then modify `pinctrl-*` properties instead.

```suggestion
&pinctrl {
	qspi_default: qspi_default {
		group1 {
			psels = <NRF_PSEL(QSPI_SCK, 0, 30)>,
				<NRF_PSEL(QSPI_IO0, 0, 29)>,
				<NRF_PSEL(QSPI_IO1, 0, 28)>,
				<NRF_PSEL(QSPI_IO2, 0, 04)>,
				<NRF_PSEL(QSPI_IO3, 0, 03)>,
				<NRF_PSEL(QSPI_CSN, 0, 31)>;
		};
	};

	qspi_sleep: qspi_sleep {
		group1 {
			psels = <NRF_PSEL(QSPI_SCK, 0, 30)>,
				<NRF_PSEL(QSPI_IO0, 0, 29)>,
				<NRF_PSEL(QSPI_IO1, 0, 28)>,
				<NRF_PSEL(QSPI_IO2, 0, 04)>,
				<NRF_PSEL(QSPI_IO3, 0, 03)>,
				<NRF_PSEL(QSPI_CSN, 0, 31)>;
			low-power-enable;
		};
	};
};

&qspi {
	pinctrl-0 = <&qspi_default_alt>;
	pinctrl-1 = <&qspi_sleep_alt>;
	pinctrl-names = "default", "sleep";
```

Reply:
OK